src/mmw/apps/core/mail/backends/boto_ses_mailer.py

```diff
@@ -0,0 +1,110 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import division
+
+import json
+import logging
+import sys
+import traceback
+import boto.ses
+
+from django.conf import settings
+from django.core.mail.backends.base import BaseEmailBackend
+
+from django_statsd.clients import statsd
+from email.mime.text import MIMEText
+
+logger = logging.getLogger(__name__)
+
+
+class EmailBackend(BaseEmailBackend):
+
+    def __init__(self, fail_silently=False, aws_region='us-east-1', *args,
+                 **kwargs):
+        super(EmailBackend, self).__init__(fail_silently=fail_silently)
+        self.mailer = BotoMailer(aws_region)
```

Comment:
The Boto function to get this is [`get_instance_metadata `](https://github.com/boto/boto/blob/2.38.0/boto/utils.py#L379), but it'll be hard to parse the dictionary that comes back without testing against an EC2 instance with real metadata.